NAS2D/Renderer/Window.h

```diff
@@ -0,0 +1,91 @@
+#pragma once
+
+#include <string>
+#include <utility>
+
+enum class WindowDisplayMode {
+	Windowed
+	,Borderless
+	,Borderless_Fullscreen
+};
```

Comment:
We use line trailing commas. Optional terminating commas (after the last list entry) are often included, particularly when list items are spilled onto multiple lines. We tend to avoid `_` in our more recent naming conventions (though there may still be examples of older uses that haven't yet renamed).

```cpp
enum class WindowDisplayMode
{
	Windowed,
	Borderless,
	BorderlessFullscreen,
};
```